objectivec/Tests/GPBWireFormatTests.m

```diff
@@ -30,9 +30,11 @@
 
 #import "GPBTestUtilities.h"
 
-#import "GPBCodedInputStream.h"
+#import "GPBProtocolBuffers.h"
```

Comment:
just curious, why import the umbrella one here instead of what you use?

Reply:
As I started fixing these up, they started needing just about everything; so rather then add everything, it seemed cleaner to just use the public everything header and the required private ones.